docs/docs/behaviors/hold-tap.md

```diff
@@ -15,11 +15,11 @@ The `tapping_term_ms` parameter decides between a 'tap' and a 'hold'.
 
 ![Simple behavior](../assets/hold-tap/case1_2.png)
 
-By default, the hold-tap is configured to also select the 'hold' functionality if another key is tapped while it's active:
+By default, the hold-tap (and mod-tap, but NOT layer-tap) are configured to also select the 'hold' functionality if another key is tapped while it's active:
```

Comment:
This should probably read "hold-tap and mod-tap (but not layer-tap) for better clarity

Reply:
> Throughout there is a use of single quotes instead of back ticks for highlighting

Happy to make this change, but I just want to confirm first that the convention is that back ticks are used for all highlighting, whether it is code-based or not. It looked to me that back ticks were for highlighting code/parameter names, whereas single quotes were used for highlighting of non-code key terms.

> This should probably read "hold-tap and mod-tap (but not layer-tap) for better clarity

My intent was to differentiate between hold-tap (which this doc is for) and mod+layer-tap through the use of the parenthesis separation. But I understand your point as well. Does your preference change at all after explaining my reasoning for the current parenthesis placement?

Reply:
> Happy to make this change, but I just want to confirm first that the convention is that back ticks are used for all highlighting, whether it is code-based or not. It looked to me that back ticks were for highlighting code/parameter names, whereas single quotes were used for highlighting of non-code key terms.

As long as it's consistent with the rest of the docs I am fine with whatever approach matches.

> My intent was to differentiate between hold-tap (which this doc is for) and mod+layer-tap through the use of the parenthesis separation. But I understand your point as well. Does your preference change at all after explaining my reasoning for the current parenthesis placement?

From a readability perspective I think trimming down the text in the parenthetical still makes sense, even if it's a little extra information as part of the main paragraph.

Reply:
Quite honestly there's already some inconsistency in the documentation around this.  Until someone aligns all the pages I think what you've done is fine.